guides/common/modules/proc_enabling-the-project-client-name-repository-for-rhel-9-and-rhel-8.adoc

```diff
@@ -0,0 +1,55 @@
+[id="enabling-the-project-client-name-repository-rhel-9-and-rhel-8"]
+= Enabling the {project-client-name} repository for {RHEL} 9 and {RHEL} 8
+
+To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
+
+* xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[]
+* xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[]
+
+ifeval::["{mode}" == "disconnected"]
+.Prerequisites
+* Ensure that you import all content ISO images that you require into {ProjectServer}.
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Red Hat Repositories*.
+. In the Available Repositories pane, enable the *Recommended Repositories* to get the list of repositories.
+. Click *{Team} {project-client-name} for RHEL 9 x86_64 (RPMs)* or *{Team} {project-client-name} for RHEL 8 x86_64 (RPMs)* to expand the repository set.
+. For the *x86_64* architecture, click the *+* icon to enable the repository.
++
+If the *{project-client-name}* items are not visible, it may be because they are not included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
```

Comment:
same as above

Reply:
Updated capitalization of 'RH subscription manifest'.